examples/react-apollo/cypress/e2e/4-authenticated/token-refresh.cy.ts

```diff
@@ -0,0 +1,16 @@
+import faker from '@faker-js/faker'
+
+context('Token refresh', () => {
```

Comment:
Could you please add tests for the case where the current token expires? (e.g: it's still in local storage, but the user has not visited the page for 4 weeks)